modules/pipeline/providers/edgepipeline_register/interface.go

```diff
@@ -40,6 +40,8 @@ type Interface interface {
 
 	ShouldDispatchToEdge(source, clusterName string) bool
 	GetEdgeBundleByClusterName(clusterName string) (*bundle.Bundle, error)
+	OnEdge(func(context.Context))
+	OnCenter(f func(context.Context))
```

Comment:
- please unify the format
- add comment for each method
  - can invoke multiple times?
  - sync or async?
  - others

Reply:
adjusted and added